ramls/loan.json

```diff
@@ -21,6 +21,10 @@
       "description": "ID of the item lent to the patron",
       "type": "string"
     },
+    "itemLocationId": {
+      "description": "The effective location, at the time of checkout, of the item loaned to the patron. This is a UUID that refers to /locations/{id}.",
+      "type": "string"
```

Comment:
(new) UUID properties typically use a regular expression for validation.

The [current chosen regular expression](https://dev.folio.org/guides/uuids/) is `^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[1-5][a-fA-F0-9]{3}-[89abAB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$`